2_pvc_destination_gen/pvc-destination-gen.yml

```diff
@@ -5,36 +5,21 @@
     - vars/storage-class-mappings.yml
   tasks:
     - import_tasks: ../common/assert-kubeconfig.yml
+      tags:
```

Comment:
I'm curious: how do you decide when to write two separate playbooks vs writing one and always running it with particular tags? This seems to be a pattern in CI as well. Is the main motivation to avoid duplication of tasks?

Reply:
@djwhatle Yes. DRYer code, plus, applying tags on included tasks is easier. Tag applied on `include_task` will be applied to all the tasks included from the file.